docs/usage.rst

```diff
@@ -9,7 +9,7 @@ The `ansible-rulebook` CLI supports the following options:
     usage: ansible-rulebook [-h] [-r RULEBOOK] [-e VARS] [-E ENV_VARS] [-v] [--version] [-S SOURCE_DIR] [-i INVENTORY] [-W WEBSOCKET_URL] [--websocket-ssl-verify WEBSOCKET_SSL_VERIFY]
                             [--websocket-token-url WEBSOCKET_TOKEN_URL] [--websocket-access-token WEBSOCKET_ACCESS_TOKEN] [--websocket-refresh-token WEBSOCKET_REFRESH_TOKEN]
                             [--id ID] [-w] [-T PROJECT_TARBALL] [--controller-url CONTROLLER_URL] [--controller-token CONTROLLER_TOKEN] [--controller-ssl-verify CONTROLLER_SSL_VERIFY]
-                            [--print-events] [--heartbeat n] [--execution-strategy sequential|parallel] [--skip-audit-events]
+                            [--print-events] [--heartbeat n] [--execution-strategy sequential|parallel] [--skip-audit-events] [--vault-password-file] [--vault-id] [--ask-vault-pass]
```

Comment:
@bzwei There is a section below which provides the details for all the cmd line options that is missing